handbook/people-ops/benefits-and-perks.md

```diff
@@ -2,13 +2,17 @@
 
 We want our teammates to be happy, healthy, and productive.
 
+## 🏡 All remote
+
+We are an [all-remote team](../../company/remote/index.md) so you never waste time commuting again, have flexible work hours, and work with the best people worldwide. We give you a [budget to either work from a coworking space or setup your home office](spending-company-money.md).
+
 ## 🏖️ Vacation
 
 We have a flexible vacation policy with mandatory annual vacation time that encourages our team to recharge when they need to. Read more about our philosophy on [mandatory vacation time](https://about.sourcegraph.com/blog/why-vacation-at-tech-companies-should-be-mandatory-better-code-happier-people).
 
 ## 🏥 Medical
 
-We offer high-quality medical and dental care for all full-time employees and their dependents.
+We offer high-quality medical and dental care for all full-time US employees and their dependents. If you are a full-time teammate located outside of the US, your salary will be increased instead.
```

Comment:
@mercadon to confirm the details here.

Reply:
As far as I am concerned, this isn't a policy change, it just adds documentation to what we have been doing for a while. See my comment here: https://github.com/sourcegraph/about/pull/552#discussion_r375911778

I would rather get this merged so if this addition to the docs needs confirmation, then I can just delete it.